plaso/output/l2t_csv.py

```diff
@@ -123,19 +156,12 @@ def _GetOutputValues(self, event):
     extra_attributes = '; '.join(extra_attributes)
     extra_attributes = extra_attributes.replace('\n', '-').replace('\r', '')
 
-    inode = getattr(event, 'inode', None)
-    if inode is None:
-      if hasattr(event, 'pathspec') and hasattr(
-          event.pathspec, 'image_inode'):
-        inode = event.pathspec.image_inode
-    if inode is None:
-      inode = '-'
-
-    hostname = self._FormatHostname(event)
-    username = self._FormatUsername(event)
+    inode = self._FormatInode(event_data)
+    hostname = self._FormatHostname(event_data)
+    username = self._FormatUsername(event_data)
 
     notes = []
-    note_string = getattr(event, 'notes', None)
+    note_string = getattr(event_data, 'notes', None)
```

Comment:
I don't think this is ever used, I don't see "notes" being populated on either the event or the event data.

Reply:
Out of scope of this PR but I've created https://github.com/log2timeline/plaso/issues/2534